src/Cryptography/Sodium/Bindings/Signing.hs

```diff
@@ -97,49 +106,52 @@ foreign import capi "sodium.h crypto_sign_seed_keypair"
 -- Combined Mode --
 -------------------
 
--- | The 'cryptoSign' function prepends a signature to a message @m@, whose length is @mlen@ bytes,
--- using the secret key @secret key@.
+-- | Prepend a signature to a message, using the secret key.
 --
 -- The signed message, which includes the signature plus an unaltered copy of the message, is put
--- into @signed message@ and is 'cryptoSignBytes' + @mlen@ bytes long.
+-- into the signed message buffer, and is of length 'cryptoSignBytes' + @length of the message@ bytes.
 --
 -- If the pointer to the length of the signed message is not a 'Foreign.nullPtr',
 -- then the actual length of the signed message is stored in it.
 --
+-- /See:/ [crypto_sign()](https://doc.libsodium.org/public-key_cryptography/public-key_signatures#combined-mode)
+--
 -- @since 0.0.1.0
 foreign import capi "sodium.h crypto_sign"
   cryptoSign ::
-    -- | Pointer to the @signed message@.
+    -- | Pointer to the signed message.
     Ptr CUChar ->
-    -- | Pointer to the length of the @signed message@.
+    -- | Pointer to the length of the signed message.
     Ptr CULLong ->
-    -- | Pointer to the @message@ to sign.
+    -- | Pointer to the message to sign.
     Ptr CUChar ->
-    -- | Length of the @message@.
+    -- | Length of the message.
     CULLong ->
-    -- | Pointer to the @secret key@.
+    -- | Pointer to the secret key.
     Ptr CUChar ->
     -- | Return code is 0 on success, -1 on error.
     IO CInt
 
--- | The 'cryptoSignOpen' function checks that the signed message @signed message@,
--- whose length is @smlen@ bytes, has a valid signature for the public key @public key@.
+-- | Check that the signed message has a valid signature for the public key.
 --
--- On success, it puts the message without the signature into @m@, stores its length in @mlen@
--- if @mlen@ is not a 'Foreign.nullPtr' pointer.
+-- On success, it puts the message without the signature into, and stores its length in
+-- the buffer holding the length of the message, if the pointer is not a 'Foreign.nullPtr'.
```

Comment:
This sentence is a bit unclear. It puts the message without the signature into what?